content/docs/develop/operate/k8s_support/index.md

```diff
@@ -20,13 +20,16 @@ Keptn versions are expressed as `x.y.z`, where `x` is the major version, `y` is
 <!-- use https://www.tablesgenerator.com/markdown_tables# for editing -->
 
 | Keptn Version /<br>Installation       | Kubernetes      | AKS             | EKS         | GKE             | OpenShift | Minikube              | MicroK8s<br>(experimental) | Minishift<br>(experimental) |
-|---------------------------------------|-----------------|-----------------|-------------|-----------------|-----------|-----------------------|----------------------------|-----------------------------|
-| 0.6.x / <br>Control & Execution Plane | 1.15.x - 1.13.x | 1.15.x (1)   | 1.15 - 1.14 | 1.15.x - 1.14.x | 3.11      | -                     | -                          | 1.34.2<br>(K8s: 1.11.x)     |
-| 0.6.x / <br>Control Plane             | 1.16.x - 1.13.x | 1.16.x - 1.15.x | 1.15 - 1.14 | 1.15.x - 1.14.x | 3.11      | 1.2 <br>(K8s:1.15.x) | 1.18.x                     | 1.34.2<br>(K8s: 1.11.x)     |
+|---------------------------------------|:-----------------:|:-----------------:|:-------------:|:-----------------:|:-----------:|:-----------------------:|:----------------------------:|:-----------------------------:|
+| **0.7.x** / <br>Control & Execution Plane | 1.18.x - 1.14.x | 1.16.x - 1-15.x (1)   | 1.16 - 1.14 | 1.16.x - 1.14.x | 3.11      | -                     | -                          | 1.34.2<br>(K8s: 1.11.x)     |
+| **0.7.x** / <br>Control Plane             | 1.18.x - 1.14.x | 1.16.x - 1.15.x (1)   | 1.16 - 1.14 | 1.16.x - 1.14.x | 3.11      | 1.4 <br> (K8s:1.16)<br>1.3.1,1.2 <br>(K8s:1.15.x) | 1.18.x                     | 1.34.2<br>(K8s: 1.11.x)     |
+| 0.6.x / <br>Control & Execution Plane | 1.15.x - 1.13.x | 1.15.x (1)   | 1.15 - 1.14 (2) | 1.15.x - 1.14.x | 3.11      | -                     | -                          | 1.34.2<br>(K8s: 1.11.x)     |
+| 0.6.x / <br>Control Plane             | 1.16.x - 1.13.x | 1.16.x - 1.15.x (1)  | 1.15 - 1.14 (2) | 1.15.x - 1.14.x | 3.11      | 1.2 <br>(K8s:1.15.x) | 1.18.x                     | 1.34.2<br>(K8s: 1.11.x)     |
 
 **Remarks:**
 
 * (1): AKS with K8s version before 1.15 might fail [#1429](https://github.com/keptn/keptn/issues/1429), due to a known AKS issue: [#69262](https://github.com/kubernetes/kubernetes/issues/69262)
+* (2): EKS and GKE do not provide K8s 1.16 clusters (checked on: 9th April, 2020)
```

Comment:
on EKS, 1.16 is available (checked May 13th) - see https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html .
On GKE, 1.16 is not available right now, but should be made available "within the next weeks".

In addition, I think we should separate the the footnotes for 0.6 and 0.7. At the time of releasing 0.6.x, EKS and GKE did not provide 1.16 (so that's fine with April 9th). 

However, for 0.7 we need to be up-to-date and have a seperate footnote with my findings.

Reply:
True, we need to keep 0.6. separated from 0.7. Please take a look at my changes.